configs/altogether.yaml

```diff
@@ -0,0 +1,53 @@
+trainer:
+  precision: bf16-mixed
+  logger:
+  - class_path: lightning.pytorch.loggers.CSVLogger
+    init_args:
+      save_dir: /Users/Shinji/models
+  - class_path: lightning.pytorch.loggers.WandbLogger
+    init_args:
+      save_dir: /Users/Shinji/models
```

Comment:
What's with this user profile?

Reply:
Oh goddamnit. Why do you pilot the Yoyodyne Kyle?

Nah but let's clean these ups for non-otaku user directories. (e.g. ~/Documents should be neutral enough)